build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -4,12 +4,26 @@
   <PropertyGroup>
     <_SourceTopDir>..\..</_SourceTopDir>
     <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MandroidInternalDir>$(OutputPath)\lib\mandroid-internal</_MandroidInternalDir>
```

Comment:
it is placed in the mandroid-internal directory because it is not remapped yet. monodroid remaps it and put remapped mdoc in the mandroid directory

that will change once we move the remap tool to xamarin-android repo